### ISSUE TITLE
FluiddPi 'unknown' error, no communication with klippy

### ISSUE THREAD

**Issue by bulbersator** [fluidd-core/FluiddPI#18]
Hi, I installed FluidPi on RPI Zero 2W, and I have 'unknown' error. 
On first run I had error about printer.cfg file not present, when I pasted my config file (that was working on the same printer under Octoprint) it changed to 'unknown' error are there seem to be no communication with klippy service. 
It looks like this error is repeating:
`[websockets.py:build_error()] - JSON-RPC Request Error: -32601`
I installed FluidPi once again from ground up and the behavior is the same. 
I also tried to update everything - nothing changes. 
![obraz](https://user-images.githubusercontent.com/90289687/145094888-789a0999-4a9a-4eff-977f-d2b2d1ca6679.png)

I'm attaching log files: 
[klippy.log.txt](https://github.com/fluidd-core/FluiddPI/files/7671102/klippy.log.txt)
[moonraker.log-1.txt](https://github.com/fluidd-core/FluiddPI/files/7671103/moonraker.log-1.txt)
I thought maybe the configuration file had an error and I added one entry at a time, but when there are no more errors with the configuration file, an unknown error appears.

edit: I also tried installing with KIAUH and the error is the same.

**Comment by BeanieBen9990** [comment]
I'm having this same issue, I have no idea why my configuration wouldn't be working in fluidd but worked fine in octoprint

**Comment by BeanieBen9990** [comment]
Ah nevermind, I fixed the problem. I'm not sure if the same steps will work for you, but I made sure to chown everything in the /home/pi directory to pi:pi and then I rebooted. This seemed to fix the problem, but I had the problem again when I turned off the printer and then turned it back on, but was fixed when I ssh'd into the pi and rebooted, so it seems like the problem might arise from either klipper, fluidd, or moonraker booting up before the printer's mainboard does.
